doc/man7/provider-signature.pod

```diff
@@ -234,6 +300,34 @@ is I<tbslen> bytes long.
 The signature is pointed to by the I<sig> parameter which is I<siglen> bytes
 long.
 
+=head2 Message Verify Functions
+
+These functions are suitable to algorithms that implement algorithms that
+accumulates a full message and signs the result of that accumulation, such as
```

Comment:
These are the verify functions - so why are we talking about signing here?

Reply:
Changed